ibaqpy/bin/peptides2protein.py

```
@@ -0,0 +1,307 @@
+import math
+import pandas as pd
+import numpy as np
+from ibaqpy.data.data import histones
+from matplotlib.backends.backend_pdf import PdfPages
+from pandas import DataFrame, Series
+from pyopenms import *
```

Ruff (0.8.0)

`from pyopenms import *` used; unable to detect undefined names

(F403)